mlcomp/parallelm/ml_engine/sagemaker_engine.py

```diff
@@ -129,9 +130,13 @@ def _setup_env(self, eng_args_config):
         os.environ[SageMakerEngine.AWS_SECRET_ACCESS_KEY] = aws_secret_access_key
 
     def _setup_logger(self, eng_args_config):
-        logging_level_name = EeArg(eng_args_config.get('logging_level',)).value
-        logging_level = constants.LOG_LEVELS.get('info' if not logging_level_name else logging_level_name.lower())
-        boto3.set_stream_logger('boto3.resources', logging_level)
+        modules_logging_level = EeArg(eng_args_config.get('boto3_logging_level')).value
```

Comment:
I know this is not part of this fix, but can the "boto3.resourses" be a constant?

Reply:
It is not a string that we ever change and belongs to sagemaker. In such case, as well as others, it is more readable to see it directly in the code.